pallets/ethereum-events/src/lib.rs

```diff
@@ -422,8 +422,11 @@ pub mod pallet {
 
     #[pallet::call]
     impl<T: Config> Pallet<T> {
-        /// This extrinsic is being deprecated. Use add_ethereum_log
         // We need to maintain this till SYS-888 is resolved. After that it can be removed.
+        #[deprecated(
+            since = "5.5.0",
+            note = "This extrinsic is being deprecated. Use add_ethereum_log"
```

Comment:
Will this cause a circular reference :smile:? Since add_ethereum_log is also being deprecated? Maybe we should have the same deprecation note as add_ethereum_log?

Reply:
I can update the message, to reflect that they are both deprecated